android/tangram/src/com/mapzen/tangram/FontFileParser.java

```diff
@@ -135,7 +135,14 @@ private void processDocument(XmlPullParser parser) throws XmlPullParserException
                                 fallbackFontDict.put(weight, new Vector<String>());
                             }
 
-                            fallbackFontDict.get(weight).add(filename);
+                            // Don't use UI fonts
+                            if (filename.indexOf("UI-") >= 0) {
+                                continue;
+                            }
+
+                            String fullFileName = "system/fonts/" + filename;
```

Comment:
We can probably add a constant for `/system/fonts/` and use that constant everywhere in this file.